test/toolchain/BUILD.bazel

```diff
@@ -1,3 +1,19 @@
+load("@bazel_skylib//rules:build_test.bzl", "build_test")
 load(":toolchain_test.bzl", "toolchain_test_suite")
 
 toolchain_test_suite(name = "toolchain_test_suite")
+
+genrule(
```

Comment:
Can you use `run_binary` instead? I hate genrule lol

Reply:
I don't think so, because `run_binary` requires that `argv[0]` is the tool, but we don't have a single tool here we have a toolchain we need to expand it from `$(RUSTC)`?

Reply:
You’re right, never mind!